tests/functional/aws-node-sdk/lib/utility/bucket-util.js

```diff
@@ -17,7 +17,8 @@ class BucketUtility {
 
     bucketExists(bucketName) {
         return this.s3
-            .headBucket({ Bucket: bucketName }).promise()
+            .headBucket({ Bucket: bucketName })
+            .promise()
```

Comment:
These changes are unrelated to this PR and will likely conflict with Maha's, work, can we drop this? this will be handled by a prettier later